products/pluto/pluto_build/00_setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+source ./bash/config.sh
+set_error_traps
+
+# DROP all tables
+if [[ $1 == "drop" ]]; then
```

Comment:
since this file only drops all tables, should we just get rid of this if clause

Reply:
@fvankrieken Good call! Updated